api/openapi-spec/swagger.json

```diff
@@ -15495,6 +15495,11 @@
           "kind": "DeleteOptions",
```

Comment:
There should be a large change to this file that includes the new v2 schema definitions for HPA. That means the autoscaling/v2 API is not actually being served.

Reply:
you need to enable the new group/version in pkg/controlplane/instance.go#DefaultAPIResourceConfigSource() to make it be served, then regenerate this file and add a new test fixture to test/integration/etcd/data.go to exercise the new version of HPA

Reply:
this also means that there are no integration or e2e tests exercising creating/updating HPAs using the new API version... there needs to be at least minimal coverage of that

Reply:
Ok, Thanks.